src/lib.rs

```diff
@@ -203,10 +202,18 @@ impl<'data> Iterator for DataFormatDescriptorIterator<'data> {
     }
 }
 
-struct KeyValueDataIterator<'data> {
+/// An iterator that parses the key-value pairs in the KTX2 file.
+pub struct KeyValueDataIterator<'data> {
     data: &'data [u8],
 }
 
+impl<'data> KeyValueDataIterator<'data> {
+    /// Create a new iterator.
```

Comment:
We should document that this takes the key-value section of a KTX2 file, not e.g. the entire file.